renovate/default-config.json

```diff
@@ -1,26 +1,48 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:base",
+    "config:best-practices",
+    "schedule:automergeWeekdays",
     "schedule:monthly",
+    ":assignAndReview(team:mirsg)",
+    ":automergeBranch",
     ":disableDependencyDashboard",
     ":enablePreCommit",
-    ":noUnscheduledUpdates"
+    ":label(renovate)",
+    ":noUnscheduledUpdates",
+    ":prHourlyLimitNone"
   ],
   "commitMessageAction": "Renovate:",
-  "reviewers": ["team:mirsg"],
   "packageRules": [
     {
-      "description": "Automatically merge minor and patch-level updates",
+      "description": "Automatically merge, minor and patch-level updates",
       "automerge": true,
-      "automergeType": "branch",
       "matchManagers": ["github-actions", "pre-commit"],
       "matchUpdateTypes": ["digest", "minor", "patch"]
     },
     {
       "description": "Shorten commit titles",
       "commitMessageTopic": "{{depName}}",
       "matchManagers": ["github-actions", "pre-commit"]
+    },
+    {
+      "description": "Combine action artefact updates together",
+      "groupName": "artefacts",
+      "matchDepNames": ["actions/download-artifact", "actions/upload-artifact"]
+    },
+    {
+      "description": "Only allow major updates",
+      "enabled": false,
+      "matchDepNames": ["renovatebot/pre-commit-hooks"],
+      "matchUpdateTypes": ["minor", "patch", "pin"]
+    },
```

Comment:
Annoying when updates frequently

Reply:
just wondering, does this apply to our MIRSG meta-hook, or the individual hooks (or both)?

Reply:
This would apply to every time we use `renovatebot/pre-commit-hooks`. Currently, I've not added it to the `mirsg-hooks`, because it is very slow. It is only added at the `.pre-commit-config.yaml` of this repo. However, if a repo were to use it, then it will only update on major versions. So at least you don't have to wait for it to update frequently (which takes ages).